tests/spec/auth/api.js

```diff
@@ -193,6 +193,62 @@ define([
           });
       });
     });
+
+    bdd.describe('bestChoice', function () {
+      var config;
+      var ui;
+      var windowMock;
+      var client;
+
+      bdd.beforeEach(function () {
+        windowMock = new WindowMock();
+        ui = new LightboxBroker('client_id', {
+          window: windowMock
+        });
+        config = {
+          ui: ui,
+          state: 'state',
+          redirectUri: 'http://redirect.to.me',
+          scope: 'profiles'
+        };
+
+        client = new FxaRelierClient('client_id', {
+          window: windowMock
+        });
+      });
+
+      bdd.it('creates and loads a UI', function () {
+        sinon.stub(ui, 'bestChoice', function () {
+          return p();
+        });
+
+        return client.auth.bestChoice(config);
```

Comment:
I realize other tests do not do it, I feel like we should ensure `ui.bestChoice` has actually been called.